upstairs/src/downstairs.rs

```diff
@@ -3548,7 +3548,8 @@ impl Downstairs {
     }
 
     #[cfg(test)]
-    /// Create a test downstiars which has one downstairs client transitioned to LiveRepair
+    /// Create a test downstiars which has one downstairs client transitioned to
```

Comment:
```suggestion
    /// Create a test downstairs which has one downstairs client transitioned to
```

Reply:
(not a new typo, but I saw it while reviewing)

Reply:
I went and found and replaced all the downstiars cases.